e2e-tests/README.md

```diff
@@ -60,6 +114,50 @@ sed -i -e 's^perconalab^172.30.162.173:5000/namespace^' e2e-tests/functions e2e-
 ./e2e-tests/storage/run
 ./e2e-tests/self-healing/run
 ./e2e-tests/operator-self-healing/run
-```
-if test failed, rerun it at least 3 times
-*NB*: each test creates own namespace and doesn't cleanup objects in case of failure
+....
+```
+
+## Using environment variables to customize the testing process
+
+### Re-declaring default image names
+
+You can use environment variables to re-declare all default docker images used for testing. The
+full list of variables is the following one:
+
+* `IMAGE_PXC` - Percona XtraDB Cluster, `perconalab/percona-xtradb-cluster-operator:main-pxc8.0` by default,
+* `IMAGE_PMM` - Percona Monitoring and Management (PMM) client, `perconalab/pmm-client:dev-latest` by default,
+* `IMAGE_PROXY` - ProxySQL, `perconalab/percona-xtradb-cluster-operator:main-proxysql` by default,
+* `IMAGE_HAPROXY` - HA Proxy, `perconalab/percona-xtradb-cluster-operator:main-haproxy` by default,
+* `IMAGE_BACKUP` - backups, `perconalab/percona-xtradb-cluster-operator:main-pxc8.0-backup` by default,
+* `IMAGE_LOGCOLLECTOR` - Log Collector, `perconalab/percona-xtradb-cluster-operator:main-logcollector` by default,
+
+### Running the Operator cluster-wide
+
+Also, you can run the Operator for the tests in a [cluster-wide mode](https://www.percona.com/doc/kubernetes-operator-for-pxc/cluster-wide.html) if needed. This feature is turned on if the following variable is declared:
+
+```
+export OPERATOR_NS=pxc-operator
+```
+
+### Using automatic clean-up after testing
+
+By default, each test creates its own namespace and does not clean up objects in case of failure.
+
+To avoid manual deletion of such leftovers, you can run tests on a separate cluster and use the following environment variable to make the ultimate clean-up:
+
+```
+export CLEAN_NAMESPACE=1
+```
+
+**Note:** this will cause **deleting all namespaces** except default and system ones!
+
+### Skipping backup tests on S3-compatible storage
+
+Making backups [on S3-compatible storage](https://www.percona.com/doc/kubernetes-operator-for-pxc/backups.html#making-scheduled-backups) needs creating Secrets to have the access to the S3 buckets. There is an environment variable which allows to skip all tests which require such Secrets.
+
+```
+SKIP_BACKUPS_TO_AWS_GCP=1
```

Comment:
I am going to enable it by default, so please mention that it is enabled by default